src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const categories = [];
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Kindly make use of the duck pattern to define the action type check_status. e.g
``` javascript
  const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```